webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -168,9 +168,12 @@ const ResourcePoolCard: React.FC<Props> = ({ resourcePool: pool }: Props) => {
               <div className={css.name}>{pool.name}</div>
             </div>
             <div className={css.default}>
-              {(pool.defaultAuxPool && pool.defaultComputePool && <span>Default</span>) ||
-                (pool.defaultComputePool && <span>Default Compute</span>) ||
-                (pool.defaultAuxPool && <span>Default Aux</span>)}
+              <span>
+                {defaultLabel ||
+                  (pool.defaultAuxPool && pool.defaultComputePool && 'Default') ||
+                  (pool.defaultComputePool && 'Default Compute') ||
+                  (pool.defaultAuxPool && 'Default Aux')}
+              </span>
```

Comment:
If we are changing this part, I prefer passing something like `descriptiveLabel` and move relevant computation to the upper level, so that for this part to be empty, I don't have to pass `defaultLabel=" "` any more

Reply:
I agree about not wanting to pass in `" "`, but I prefer handling as much of the computation in the ResourcePoolCard level as we can. My reasoning for that is that encapsulating the "business logic" in the component makes it easier to just drop it into a page, rather than requiring us to duplicate computations. This is your PR and I don't think it's a big deal though, so up to you.